src/ndjson.rs

```diff
@@ -24,8 +26,26 @@ impl Log for Logger {
             let stdout = io::stdout();
             let mut handle = stdout.lock();
             let level = get_level(record.level());
-            let time = time::UNIX_EPOCH.elapsed().unwrap().as_millis();
-            write!(&mut handle, "{{\"level\":{},\"time\":{},\"msg\":", level, time).unwrap();
+            #[cfg(not(readable_timestamp))]
+            {
+                let time = time::UNIX_EPOCH.elapsed().unwrap().as_millis();
+                write!(
+                    &mut handle,
+                    "{{\"level\":{},\"time\":{},\"msg\":",
+                    level, time
+                )
+                .unwrap();
+            }
+            #[cfg(readable_timestamp)]
+            {
+                let time = Local::now().format("%Y-%m-%d %H:%M:%S%.f%z");
```

Comment:
I think this should be `Local::now().to_rfc3339()` or `Local::now().to_rfc3339_opts(SecondsFormat::Millis, true)`. iso 8601+3339 is widely used in json logs. See https://docs.rs/chrono/latest/chrono/struct.DateTime.html?search=#method.to_rfc3339_opts 

Personally, I'd prefer this to be in UTC anyway because I don't like to depend on a server's timezone.